doc/tutorial/printing_drawing.txt

```diff
@@ -14,6 +14,15 @@ compilation.  These two functions print expression graphs in different ways:
 Theano also provides :func:`pydotprint` that creates a *png* image of the function.
 You can read about them in :ref:`libdoc_printing`.
 
+.. note::
```

Comment:
Actually, there could be another small change here that I missed. (added "will")

```
.. note::

    When printing Theano functions, they can sometimes be hard to
    read.  To help with this, you can disable some Theano optimizations
    by using the Theano flag:
    ``optimizer_excluding=fusion:inplace``. Do not use this during
    real job execution, as this will make the graph slower and use more
    memory.
```